src/style.css

```diff
@@ -1 +1,114 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir-Book;
+  src: local('Avenir-Book'), url('fonts/Avenir-Book.ttf') format('ttf');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: local('Avenir-Heavy'), url('fonts/Avenir-Heavy.ttf') format('ttf');
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+input {
+  border: none;
+  outline: none;
+}
+
+input[type="text" i] {
+  padding-block: 0;
+  padding-inline: 0;
+}
+
+body {
+  margin: 0;
+  font-family: Avenir-Book, sans-serif;
+  font-size: 18px;
+  color: #3D4E61;
+}
+
+form[data-qa="big"] {
```

Comment:
No need to use attributes as css selector, use classes instead. Fix it everywhere